src/apple/project.rs

```diff
@@ -95,6 +95,7 @@ pub fn gen(
         |map| {
             map.insert("file-groups", &source_dirs);
             map.insert("ios-frameworks", metadata.ios().frameworks());
+            map.insert("ios-vendor-frameworks", metadata.ios().vendor_frameworks());
```

Comment:
What about macOS vendor frameworks?

Reply:
I didn't address it since I don't currently have a test for it, but it would be trivial to add and would behave exactly as the ios one.

Reply:
Since it's possible for the user to specify macOS ones, it'd be surprising if we ignored them (the same applies to asset catalogs, actually)

since it's trivial + just gets fed into XcodeGen, not having a test for them is probably fine.